src/rt-app_parse_config.c

```diff
@@ -838,6 +904,15 @@ parse_thread_data(char *name, struct json_object *obj, int index,
 		assure_type_is(phases_obj, obj, "phases",
 					json_type_object);
 
+		/*
+		 * Taskgroup should only be specified on thread level if the thread
+		 * does not contain one or more phases.
```

Comment:
Isn't a bit confusing?
What about "Taskgroup must not be specified at thread level if phases are present".

Reply:
I changed it to your sentence.